lib/sites.ts

```diff
@@ -52,6 +53,11 @@ export function useSiteData(siteUUID: string) {
     actualsFetcher
   );
 
+  const { data: inverterData, error: inverterError } = useSWR(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL_GET}/sites/${siteUUID}/inverters`,
+    invertersFetcher
```

Comment:
If you don't supply the fetcher here, SWR will use the default one (that we supplied in `_app.tsx`). It performs the same logic, so using the default is ok.
```suggestion

```